tests/test_other.py

```diff
@@ -710,6 +710,10 @@ def test_cmake_static_lib(self, custom):
     else:
       self.assertTrue(building.is_ar('libstatic_lib.a'))
 
+  # Tests that cmake functions which require evaluation via the node runtime run properly with pthreads
+  def test_cmake_pthreads(self):
+    self.run_process([EMCMAKE, 'cmake', '-DCMAKE_C_FLAGS_INIT=-pthread', test_file('cmake/target_js')])
```

Comment:
Why not just use `CMAKE_C_FLAGS` here?    `CMAKE_C_FLAGS_INIT` looks like something that is more often used in toolchain files (although I am not too familiar with it).

Reply:
yeah, good catch